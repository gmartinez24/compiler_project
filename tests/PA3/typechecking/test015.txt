// Devan Shea
// Testing for calling wrong type for function
main

bool x = true;

{
    call printInt(x);
}.