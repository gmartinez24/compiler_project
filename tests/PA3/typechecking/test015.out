Error type-checking file.
TypeError(6,5)[Call with args TypeList(bool) matches no function signature.]